apps/src/util/curriculumRecommender/curriculumRecommender.js

```diff
@@ -0,0 +1,161 @@
+import {
+  IMPORTANT_TOPICS,
+  UTC_PUBLISHED_DATE_FORMAT,
+  FAKE_RECOMMENDER_SCORING,
+} from './curriculumRecommenderConstants';
+import moment from 'moment';
+
+const now = moment().utc();
+const oneYearAgo = moment()
+  .utc()
+  .year(now.year() - 1);
+const twoYearsAgo = moment()
+  .utc()
+  .year(now.year() - 2);
+
+/*
+ * Curriculum recommenders: Each recommender receives an array of all curricula to consider and any preferences it should prioritize. The given
+ * recommender will score each curriculum based on the questions associated with that recommender. If a curriculum satisfies the conditions of a
+ * question, it receives points based on that recommender's scoring framework found in 'curriculumRecommenderConstants.js'. Once all curricula
+ * have been scored, the recommender returns an array of the passed-in curricula sorted in descending order by score (ties are broken by
+ * prioritizing featured curricula and more recently published curricula).
+ */
+export const getTestRecommendations = (
+  curricula,
+  duration,
+  marketingInitiative,
+  schoolSubjects,
+  csTopics
+) => {
+  const curriculaScores = [];
+  curricula.forEach(curriculum => {
+    let score = 0;
+    score += hasDesiredDuration(FAKE_RECOMMENDER_SCORING, curriculum, duration);
```

Comment:
I was thinking it wouldn't be temporary since `getTestRecommendations` will only ever use the test scoring system. Then future recommenders will have their own function that only uses their respective scoring framework.

Reply:
ah got it -- so this function will be re-implemented for each scoring system, but the scoring functions below will be shared??

Reply:
Yep! Each scoring system uses a different subset of those scoring functions so each one will just call the ones relevant to it

Reply:
got it got it. Thanks for clarifying!